### PR TITLE
Migrate activity feed from component library

### PR DESCRIPTION
**Pull request by cgsunkel**
As part of the plan to deprecate the component library we need to migrate the specific version (v0.17.1) used by this project into its codebase. This will allow us to close the component library project.

Please note that 25,621 of the additions are in `package-lock.json` so this PR isn't as big as it looks

### CODE REVIEW

**Review by mforner13 — COMMENTED** [review]
As the activity feed is only used in find exporters for companies which aren't on Data Hub - does that mean that the card components which render data hub related objects - e.g. interactions, omis, invest projects etc. aren't used in this app, and therefore don't need to be brought in?

**Comment by cgsunkel** [comment]
> As the activity feed is only used in find exporters for companies which aren't on Data Hub - does that mean that the card components which render data hub related objects - e.g. interactions, omis, invest projects etc. aren't used in this app, and therefore don't need to be brought in?

I will remove these now

File: .gitignore [hunk 1]
```diff
@@ -65,3 +65,7 @@ cypress/
 visual-screenshots/diff
 visual-screenshots/comparison
 visual-report
+
+# Yarn version and policy
+.yarn/
+.yarnrc
```
Comment (mforner13) [inline .gitignore:71]: Missing new line :)

**Review by mforner13 — APPROVED** [review]
Looks good! 👍

**Pull request merged**